tests/StarkEx.Crypto.SDK.DifferentialTests/Utils/MathUtilsDifferentialTests.cs

```diff
@@ -0,0 +1,40 @@
+namespace StarkEx.Crypto.SDK.DifferentialTests.Utils;
```

Comment:
I see some warnings here on GitHub like:

`The file header is missing or not located at the top of the file.`  & `Prefix local calls with this` but they don't show locally not sure why. Any idea?

Reply:
Not sure, this is weird. Do you have any input here @JoseAfonsoThreeSigma ?

Reply:
Not really, I also don't have these warnings locally. We could try to add the proper rule to disable this warning to our `.editorconfig`. https://github.com/dotnet/docs/blob/main/docs/fundamentals/code-analysis/style-rules/ide0073.md

Reply:
Added `dotnet_diagnostic.IDE0073.severity = none` to the `.editorconfig` file